protocols/imap/src/main/java/org/apache/james/imap/processor/base/AbstractChainedProcessor.java

```diff
@@ -56,6 +57,11 @@ public void process(ImapMessage message, Responder responder, ImapSession sessio
         }
     }
 
+    @Override
+    public void configure(ImapConfiguration imapConfiguration) {
```

Comment:
This should configure the next element in the chain...

```
next.configure(imapConfiguration);
```

Then you should ensure all the children processor actually call super.configure(...)

Otherwise you skip configuration.